docs/cookbook/loading_molecules.rst

```diff
@@ -0,0 +1,122 @@
+Loading your data into ChemicalSystems
+======================================
+
+One of the first tasks you'll likely want to do is loading your various input files.
+In ``openfe`` the entire contents of a simulation volume, for example the ligand, protein and water is referred to
+as the :class:`openfe.ChemicalSystem`.
+
+A free energy difference is defined as being between two such :class:`ChemicalSystem` objects.
+To make expressing free energy calculations easier,
+this ``ChemicalSystem`` is broken down into various ``Component`` objects.
+It is these ``Component`` objects that are then transformed, added or removed when performing a free energy calculation.
+
+.. note::
+   Once chemical models are loaded into Components they are **read only** and cannot be modified.
+   This means that any modification/tweaking of the inputs must be done **before** the Component objects are created.
+   This is done so that any data cannot be accidentally modified, ruining the provenance chain.
+
+
+As these all behave slightly differently to accomodate their contents,
+there are specialised versions of Component to handle the different items in your system.
+We will walk through how different items can be loaded,
+and then how these are assembled to form ``ChemicalSystem`` objects.
+
+
+Loading small molecules
+-----------------------
+
+Small molecules, such as ligands, are handled using the {py:class}`openfe.SmallMoleculeComponent` class.
+These are lightweight wrappers around RDKit Molecules and can be created directly
+from an RDKit molecule:
+
+.. code::
+
+    from rdkit import Chem
+    import openfe
+
+    m = Chem.MolFromMol2File('myfile.mol2', removeHs=False)
+
+    smc = openfe.SmallMoleculeComponent(m, name='')
+
+
+.. warning::
+    Remember to include the ``removeHs=False`` keyword argument so that RDKit does not strip your hydrogens!
+
+
+As these types of structures are typically stored inside sdf files, there is a ``from_sdf_file`` convenience class method:
```

Comment:
This only reads the first entry right? We should make that clear since I'm sure someone will walk into that issue.